### DISCUSSION TITLE
Add sections / headlines to a program

### DISCUSSION THREAD

**Discussion by lsolesen** [astashov/liftosaur#197] [Ideas]
Make it possible to add sections to the the program, for instance like this:

```
# Week 1
## Day 1
### Warmup
### A-exercise
Squat / 3x8
### B-exercises
Laterals / 3x12
Face pulls / 3x12
### Shoulder exercises - add your own
```

**Comment by AdEmergency625** [comment]
If we are adding sections, I would say enabling not exercises related comments could help  more use cases.  
You could use it as a section heading.
You could use it to describe the overall program design
You could use it to note down equipment set up instructions. 
You could add any random notes like a reminder:  find a spotter before starting the follow up exercises.
